resolwe_bio/processes/support_processors/bam_conversion.py

```diff
@@ -0,0 +1,177 @@
+"""Converting BAM to BEDPE and normalised BigWig files."""
+import os
+
+from resolwe.process import (
+    Cmd,
+    DataField,
+    FileField,
+    FloatField,
+    Process,
+    SchedulingClass,
+    StringField,
+)
+
+
+class BamToBedpe(Process):
+    """Takes in a bam file and calculates a normalization factor in BEDPE format.
+
+    Done by sorting with Samtools and transformed with Bedtools.
+    """
+
+    slug = "bedtools-bamtobed"
+    name = "Bedtools bamtobed"
+    requirements = {
+        "expression-engine": "jinja",
+        "executor": {"docker": {"image": "resolwebio/rnaseq:4.11.0"}},
+        "resources": {"cores": 1, "memory": 4096},
```

Comment:
4GB of memory might not be enough for real-world sample sizes. You can test this locally though.